kyo-pure/shared/src/main/scala/kyo/aspects.scala

```diff
@@ -0,0 +1,65 @@
+package kyo
+
+object Aspects:
```

Comment:
This effect was based on `IOs` only in order to use `Locals`. It's using the new `Defers` effect now, which is similar to `IOs` but designed to provide only effect suspension without side effects.